src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1780,7 +1780,7 @@ services:
 
   form.type.sell.product.data_transformer.typeahead_redirection_target_transformer:
     class: 'PrestaShopBundle\Form\Admin\Sell\Product\DataTransformer\TypeaheadRedirectionTargetTransformer'
-    deprecated: 'The "%service_id%" service is deprecated since 1.7.9 and will be removed in next major.'
+    deprecated: 'The "%service_id%" service is deprecated since 8.0.0 and will be removed in next major.'
```

Comment:
This deprecation message has already been changed on develop https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml#L1783

So it seems like something went wrong with you rebase again, try and rebase it again based on develop maybe because this has no relation with your PR